Clicker/Views/Cards/NoResponsesCell.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  NoResponsesCell.swift
+//  Clicker
+//
+//  Created by Matthew Coufal on 10/20/18.
+//  Copyright © 2018 CornellAppDev. All rights reserved.
+//
+
+import UIKit
+
+class NoResponsesCell: UICollectionViewCell {
+    
+    // MARK: - Data vars
+    var space: CGFloat!
+    
+    // MARK: - View vars
+    var noResponsesLabel: UILabel!
+    
+    // MARK: - Constants
+    let noResponsesText: String = "No responses yet"
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        noResponsesLabel = UILabel()
+        noResponsesLabel.text = noResponsesText
+        noResponsesLabel.textColor = .clickerGrey2
+        noResponsesLabel.font = ._12MediumFont
+        noResponsesLabel.textAlignment = .center
+        contentView.addSubview(noResponsesLabel)
+        
+    }
+    
+    override func updateConstraints() {
```

Comment:
is there a reason you override this?

According to [Apple docs](https://developer.apple.com/documentation/uikit/uiview/1622512-updateconstraints) "You should only override this method when changing constraints in place is too slow, or when a view is producing a number of redundant changes.". This method will be called a bunch of times, so you are adding a ton of constraints

Reply:
That makes perfect sense. Was wondering about this actually. Thanks!